addons/project/__manifest__.py

```diff
@@ -218,6 +218,16 @@
             'project/static/src/project_sharing/search/favorite_menu/custom_favorite_item.xml',
             'project/static/src/project_sharing/**/*',
             'web/static/src/start.js',
+
+            # To use mail chatter in portal
+            'web/static/src/libs/pdfjs.js',
+            'bus/static/src/*.js',
+            'bus/static/src/services/**/*.js',
+            'bus/static/src/workers/websocket_worker.js',
+            'bus/static/src/workers/websocket_worker_utils.js',
+            ('include', 'portal.assets_chatter_helpers'),
+            'portal/static/src/chatter/core/**/*',
+            'project/static/src/chatter/**/*',
```

Comment:
I don't think we need a specific category, they should be sorted with the rest of the path from same modules.